qa/workunits/rbd/rbd_mirror_journal.sh

```diff
@@ -81,7 +81,7 @@ if [ -z "${RBD_MIRROR_USE_RBD_MIRROR}" ]; then
   wait_for_status_in_pool_dir ${CLUSTER1} ${POOL} ${image} 'up+replaying'
   wait_for_status_in_pool_dir ${CLUSTER1} ${POOL} ${image1} 'up+replaying'
 
-  all_admin_daemons ${CLUSTER1} rbd mirror stop ${POOL} ${CLUSTER2}${PEER_CLUSTER_SUFFIX}
+  all_admin_daemons ${CLUSTER1} rbd mirror stop ${POOL}
```

Comment:
Ah yes indeed I will remove this part.

Reply:
Ah, I misunderstood your change here.
The question is rather then why do you think it is a good idea not to include the peer cluster name into to the command name. That is not that I am against it. I just don't remember why we decided to have it in the first place.

Reply:
It is because of this https://github.com/ceph/ceph/pull/43359/files#diff-44db5cb9da326444610e33b842e97156e5f2d30dfcf251fda932f7031d692665L578.
As the PoolReplayer was single peer aware before the peer name was added with the pool name, now we have only one PoolReplayer for all the peers so I remove the suffix here.

Reply:
The link is wrong here the line I am talking about https://github.com/ceph/ceph/blob/c34177830a1d41cea7a19a9ff7ef1badeac4d441/src/tools/rbd_mirror/PoolReplayer.cc#L720. Before the peer name was added here as well.

Reply:
Yes I understand this. But you could just change the admin socket command name when the peer is changes, similarly how we change the name when the pool name (or image name) is changed.

As I said actually I am not against your change. Actually I don't see why we wanted the peer name there initially, and I think I like it without the peer more (unless there was some good reason we added it that I forgot).

Reply:
The code in Mirror.cc was designed to create multiple PoolReplayer for a pool if there was multiple peer so I guess that's why.